fe/fe-core/src/main/java/com/starrocks/analysis/ShowCreateTableStmt.java

```diff
@@ -47,16 +43,34 @@ public class ShowCreateTableStmt extends ShowStmt {
                     .addColumn(new Column("collation_connection", ScalarType.createVarchar(30)))
                     .build();
 
+    private static final ShowResultSetMetaData MATERIALIZED_VIEW_META_DATA =
+            ShowResultSetMetaData.builder()
+                    .addColumn(new Column("Materialized View", ScalarType.createVarchar(20)))
+                    .addColumn(new Column("Create Materialized View", ScalarType.createVarchar(30)))
+                    .build();
+
     private TableName tbl;
     private boolean isView;
+    private boolean isMaterializedView;
 
     public ShowCreateTableStmt(TableName tbl) {
-        this(tbl, false);
+        this(tbl, false, false);
     }
 
     public ShowCreateTableStmt(TableName tbl, boolean isView) {
         this.tbl = tbl;
         this.isView = isView;
+        this.isMaterializedView = false;
+    }
+
+    public ShowCreateTableStmt(TableName tbl, boolean isView, boolean isMaterializedView) {
```

Comment:
consider to use TableType to replay boolean flag.

Reply:
ok, i will change it